asf_tools/composite.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/env python
+"""Create S1 SAR Composite Mosaic using inverse area weighting ala David Small.
+
+   Path vs infiles:
+     If path is passed, code assumes files are in an ASF HyP3 RTC Stacking arrangement.
+     i.e  {path}/20*/PRODUCT/ contains the input RTC data and the area maps or
+          {path}/S1?_IW_*RTC*/ contains the input RTC data and the area maps
+         
+
+"""
+
```

Comment:
Docstrings should conform to [our (google) style](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings)